### PR TITLE
Update about the company

### PR DESCRIPTION
**Pull request by nicksnyder**
Just proposing a different phrasing that deemphasizes abstract stuff and adds details about or product.

- [x] Update other job descriptions too once finalized

### CODE REVIEW

File: job-descriptions/software-engineer.md [hunk 1]
```diff
@@ -4,10 +4,12 @@
 
 ### About the company
 
-From sustainable energy systems to self-driving cars, the innovations of the future will all rely on code. With so much software to write, developers need a better way to access, understand, and collaborate on code.
+At Sourcegraph, we are building a better, smarter foundation for software development. The innovations of the future will increasingly rely on code, so if we can make software developers more productive, we can literally bring the future sooner.
```
Comment (dadlerj) [inline job-descriptions/software-engineer.md]: What are your thoughts on:
>The innovations of the future will all rely on code. By empowering software developers today, we can literally bring the future sooner.

?

File: job-descriptions/software-engineer.md [hunk 2]
```diff
@@ -4,10 +4,12 @@
...
-From sustainable energy systems to self-driving cars, the innovations of the future will all rely on code. With so much software to write, developers need a better way to access, understand, and collaborate on code.
+At Sourcegraph, we are building a better, smarter foundation for software development. The innovations of the future will increasingly rely on code, so if we can make software developers more productive, we can literally bring the future sooner.
 
-At Sourcegraph, we are building a better, smarter foundation for software development. One that will enable developers (and the companies that rely on them) to more quickly turn great ideas into groundbreaking technology. Effectively, we want to bring the future sooner.
-Check out what we are building for developers and why it matters: https://sourcegraph.com/plan.
+We have already built the best [code search](https://about.sourcegraph.com/docs/search/) that is used and loved by large engineering organizations across the world.
+Our [browser extensions](https://about.sourcegraph.com/product/browser) provide code intelligence (e.g. hover tooltips, go to definition, find references) when viewing code in your code host.
```
Comment (dadlerj) [inline job-descriptions/software-engineer.md]: How about:
> [Sourcegraph](https://about.sourcegraph.com/product/server) and our [browser extensions](https://about.sourcegraph.com/product/browser) provide IDE-like code intelligence (e.g. hover tooltips, go to definition, find references, and much more) in your code host, in code review, and everywhere you read code. 

?

Comment (dadlerj) [reply job-descriptions/software-engineer.md]: "our" browser extensions is against the style guide, but I think it's okay here 🙂

File: job-descriptions/software-engineer.md [hunk 3]
```diff
@@ -4,10 +4,12 @@
 
 ### About the company
 
-From sustainable energy systems to self-driving cars, the innovations of the future will all rely on code. With so much software to write, developers need a better way to access, understand, and collaborate on code.
+At Sourcegraph, we are building a better, smarter foundation for software development. The innovations of the future will increasingly rely on code, so if we can make software developers more productive, we can literally bring the future sooner.
 
-At Sourcegraph, we are building a better, smarter foundation for software development. One that will enable developers (and the companies that rely on them) to more quickly turn great ideas into groundbreaking technology. Effectively, we want to bring the future sooner.
-Check out what we are building for developers and why it matters: https://sourcegraph.com/plan.
+We have already built the best [code search](https://about.sourcegraph.com/docs/search/) that is used and loved by large engineering organizations across the world.
```
Comment (dadlerj) [inline job-descriptions/software-engineer.md]: Two tweaks:

>...built the world's best...

and

>...by large and small engineering...

File: job-descriptions/software-engineer.md (hunk 1 above)
Comment (nicksnyder) [reply job-descriptions/software-engineer.md]: I feel like those two sentences should be joined together with `and` or `so`.

> The innovations of the future will all rely on code and by empowering software developers today, we can literally bring the future sooner.

Thoughts?

File: job-descriptions/software-engineer.md (hunk 2 above)
Comment (nicksnyder) [reply job-descriptions/software-engineer.md]: How about your sentence started with just "Sourcegraph's browser extensions..."?

File: job-descriptions/software-engineer.md (hunk 1 above)
Comment (dadlerj) [reply job-descriptions/software-engineer.md]: Sorry for the delay, started typing this up last week but got cut off:
===
Eh, they're two different clauses in my mind. But I'm fine putting it in one sentence. I'd prefer `so` to `and`, though — to me, it's a logic statement:

`A`: "At Sourcegraph, we are building a better, smarter foundation for software development." 
`B`: "The innovations of the future will all rely on [software development]."
`C`: "We can literally bring the future sooner"

`(A & B) -> C`

I feel that `so` captures "implies" better than `and` does.

File: job-descriptions/software-engineer.md [hunk 4]
```diff
@@ -4,10 +4,12 @@
...
+At Sourcegraph, we are building a better, smarter foundation for software development. The innovations of the future will all rely on code and by empowering software developers today we can literally bring the future sooner.
 
-At Sourcegraph, we are building a better, smarter foundation for software development. One that will enable developers (and the companies that rely on them) to more quickly turn great ideas into groundbreaking technology. Effectively, we want to bring the future sooner.
-Check out what we are building for developers and why it matters: https://sourcegraph.com/plan.
+We have already built the world's best [code search](https://about.sourcegraph.com/docs/search/) that is used and loved by large and small engineering organizations across the world.
+[Sourcegraph's browser extensions](https://about.sourcegraph.com/product/browser) provide IDE-like code intelligence (e.g. hover tooltips, go to definition, find references, and much more) in your code host, in code review, and everywhere you read code.
```
Comment (dadlerj) [inline job-descriptions/software-engineer.md]: There's this weird thing here where we imply that Sourcegraph (the server/webapp) is for code search, while the browser extensions are for code intelligence. That's why I wanted both to be in the subject of this sentence. What do you think?

**Pull request merged**